commit ec398bbd436a7b964587b8546fd65d74efb1fe24 (refs/changes/06/4706/2)
Author: Dave Eddy <dave@daveeddy.com>
Date:   2018-08-17T13:21:38-04:00 (1 year, 2 months ago)
    
    joyent/node-manta#347 mmkdir test fails when MANTA_TEST_ROLE is unset
    Reviewed by: Robert Mustacchi <rm@joyent.com>

diff --git a/test/mmkdir.test.js b/test/mmkdir.test.js
index e2063cf..7651918 100644
--- a/test/mmkdir.test.js
+++ b/test/mmkdir.test.js
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Joyent, Inc.
+ * Copyright 2018 Joyent, Inc.
  */
 
 /*
@@ -85,16 +85,15 @@ if (process.env.MANTA_TEST_ROLE) {
                   });
            });
     });
-}
-
 
-test('cleanup: rm test directory ' + TESTDIR, function (t) {
-    // Sanity checks that we don't `mrm -r` a non-test dir.
-    assert.ok(TESTDIR);
-    assert.ok(TESTDIR.indexOf('node-manta-test') !== -1);
+    test('cleanup: rm test directory ' + TESTDIR, function (t) {
+        // Sanity checks that we don't `mrm -r` a non-test dir.
+        assert.ok(TESTDIR);
+        assert.ok(TESTDIR.indexOf('node-manta-test') !== -1);
 
-    forkExecWait({ argv: [ MRM, '-r', TESTDIR ]}, function (err) {
-        t.ifError(err, err);
-        t.done();
+        forkExecWait({ argv: [ MRM, '-r', TESTDIR ]}, function (err) {
+            t.ifError(err, err);
+            t.done();
+        });
     });
-});
+}
